### PR TITLE
Release 0.10.0

### PR DESCRIPTION
**Pull request by khoroshevskyi**
- Added date filter to project annotation endpoint
- Updated peppy version (0.40.0a4) [Without attmap]
- [x] Wait for: https://github.com/pepkit/pepdbagent/pull/101

### CODE REVIEW

File: pephub/routers/api/v1/namespace.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 from pepdbagent.exceptions import ProjectUniqueNameError
 from pepdbagent.const import DEFAULT_LIMIT_INFO
 from pepdbagent.models import ListOfNamespaceInfo
+from typing import Literal
+from typing_extensions import Annotated
```
Comment (nleroy917) [inline pephub/routers/api/v1/namespace.py:12]: Drop [python < 3.9](https://fastapi.tiangolo.com/python-types/#type-hints-with-metadata-annotations) support? 🤔

**Review by nleroy917 — APPROVED** [review]
Looks good! Simple change with the convenient `pepdbagent` API

**Comment by nleroy917** [comment]
A thought: Do we want UI changes here? Or should we just wait for that? Wasn't this functionality needed for a bedbase thing? What issue num is this tagged to?

**Review by ayobi — CHANGES_REQUESTED** [review]
We don't want this in master right lol? should be 
```import { defineConfig } from 'vite'
import react from '@vitejs/plugin-react-swc'

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [react()],
})```

**Comment by nleroy917** [comment]
@ayobi What line are you referring to?

**Comment by ayobi** [comment]
this commit https://github.com/pepkit/pephub/pull/224/commits/d655625889fd7cb9689e713553517cd85849eecb, should be fixed

**Review by ayobi — APPROVED** [review]
lgtm :+1:

File: pephub/routers/api/v1/namespace.py (hunk 1 above)
Comment (khoroshevskyi) [reply pephub/routers/api/v1/namespace.py:12]: End of life for Python 3.8 in one year, so I would say let's leave it like that

**Pull request merged**